src/Laravel7Mailer.php

```diff
@@ -13,4 +13,13 @@ public function mailer($name = null)
 
         return $this;
     }
+    
+     /**
+     * Void function, bug fix for BeyondCode Hello
+     * Github issue: https://github.com/beyondcode/helo-laravel/issues/15
```

Comment:
You can tag it in the PR description. I guess

Reply:
I'm not sure what that means ?